myapp/config/application.rb

```diff
@@ -10,6 +10,9 @@ module Myapp
   class Application < Rails::Application
     # Initialize configuration defaults for originally generated Rails version.
     config.load_defaults 6.0
+    config.time_zone = 'Asia/Tokyo'
```

Comment:
一瞬、「Tokyoじゃないかな？」と思いましたがどちらでも変わらないのでOKそう！

https://moreta.github.io/ruby/rails/ruby-rails-time-date-timezone.html
```suggestion
    config.time_zone = 'Tokyo'
```